org-configurations/default-configuration.yml

```diff
@@ -208,6 +208,18 @@ client:
           WiFi Login Successful!
           You can now use the internet.
           You may leave this page open in case you want to log out.
+      user_info:
+        status:
+          text:
+            en: "Status"
+          value:
+            en: "Logged in"
+        email:
+          text:
+            en: "Email"
+        username:
+          text:
+            en: "Username"
```

Comment:
please add the mobile phone field here as well for consistency with the change requested above